### DISCUSSION TITLE
Welcome to myhelloiot Discussions!

### DISCUSSION THREAD

**Discussion by adrianromero** [adrianromero/myhelloiot#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mhaberler** [comment]
Hi Adrian,

I'll start here to post some questions a typical low-fu beginner might have

here's a bit [about what I'm trying to do](https://github.com/adrianromero/myhelloiot/issues/10#issuecomment-2151267933)

the setup is:
- "server" plattform is an ESP32/Arduino running a webserver and PicoMQTT/PicoWebsockets reporting sensor values
-  web applications are stored on-device in a LittleFS partition
- this server connects to a mobiles/pad's WiFi hotspot
- the whole thing must run sans Internet as connectivity cannot be assumed
- the webserver has a basic file management interface with editing capability sufficient to edit css/jsx etc

I guess I'll do the questions subject by subject

Michael

**Comment by tuom86** [comment]
Hi
I m very interested of this. I have few questions about this. Is this secure connect to broker, can I add some kind of login site and use this as web app and can I have buttons and sliders to send mqtt messages to broker. I would like to use this as home automation Iot platform. 
Reg Tuomo Vuorinen
Finland
